packages/cozy-client/src/associations/helpers.js

```diff
@@ -21,7 +22,10 @@ export const responseToRelationship = response =>
   })
 
 const attachRelationship = (doc, relationships) => {
-  if (doc.relationships) {
+  if (
+    doc.relationships &&
+    isEqual(Object.keys(doc.relationships), Object.keys(relationships))
```

Comment:
I don't understand this ?

Reply:
With the new dehydratation, some documents look like this:

```
{
  _id: 1,
  title: lol,
  relationships: {}
}
```

And we try to attach relationships, to it, for example:

```
{
  tasks: { data: […] }
}
```

This fails, because the original doc already has a `relationships` property… but it's incomplete.

I can add tests to this function, but I'm not even sure why the condition is there in the first place…